Sources/UIKitExtensions/UITableView+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  UITableViewFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UITableView {
+
+    func setEditing(_ editing: Bool, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UITableView: Base ) in
+            UITableView.setEditing(editing, animated: animated)
```

Comment:
This function is only useful for existing UITabBar whose isEditing change should be animated.
The isEditing Property ist writable directly so there is no need for this function.